blocks/product-content-block/index.story.jsx

```diff
@@ -24,7 +24,7 @@ const mockData = {
 };
 
 export const TitleAndContent = () => (
-	<ProductContentDisplay summary="Product Details">
+	<ProductContentDisplay summary={mockData.schema.productDetails.label}>
```

Comment:
Seemed better to get this value from the mock data provided.

Reply:
As the component used in storybook is presentational, where the value comes from doesn't have a huge impact as the component will output the value of `summary`